pytket/tests/utils_test.py

```diff
@@ -90,6 +90,13 @@ def test_all_paulis() -> None:
     assert len(list(circs)) == 3
 
 
+def test_dict_export() -> None:
+    qps1 = QubitPauliString(Qubit(0), Pauli.Y)
+    qps2 = QubitPauliString(Qubit(0), Pauli.X)
+    op = QubitPauliOperator({qps1: 1j, qps2: 0.5})
+    assert op.get_dict() == op._dict
```

Comment:
This is maybe a bit cheeky. I could do the explcit comparision if prefferred.

Reply:
Yes I think we should do an explicit comparison, to be safe against future surprises.